### ISSUE TITLE
[Bug]: 'NoneType' object is not iterable

### ISSUE THREAD

**Issue by Linnnkkk** [infiniflow/ragflow#2463]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Branch name

main

### Commit ID

c

### Actual behavior

流程开始于:
Tue, 17 Sep 2024 02:54:45 GMT
过程持续时间:
506.31 s
进度消息:
Task has been received.
Page(1~42): Extracting entities.
Page(1~42): 1/1, elapsed: 56.75517283892259s, used tokens: 27841
Page(1~42): Entities extraction progress ... 1/3
Page(1~42): 1/1, elapsed: 82.22494675405324s, used tokens: 33271
Page(1~42): Entities extraction progress ... 2/3
Page(1~42): 1/1, elapsed: 145.8512048800476s, used tokens: 17617
Page(1~42): Entities extraction progress ... 3/3
Page(1~42): Extracting community reports.
Page(1~42): Communities: 1/2, elapsed: 10.652305909898132s, used tokens: 3229
Page(1~42): Communities: 2/2, elapsed: 22.517052062787116s, used tokens: 6454
Page(1~42): Extracting mind graph.
Page(1~42): Finished slicing files(90). Start to embedding the content.
Page(1~42): [ERROR]Embedding error:'NoneType' object is not iterable
Page(1~42): Finished embedding(1.95)! Start to build index!

### Steps to reproduce

```Markdown
解析失败
```

**Comment by KevinHuSh** [comment]
What is your embedding model?

**Comment by Linnnkkk** [comment]
> What is your embedding model?

siliconflow

**Comment by ai408** [comment]
I also meet this problem with siliconflow.

**Comment by yuzhichang** [comment]
@ai408 Please upgrade to v0.14.1 which has much better logging. `task_executor_0.log` shall contain the detail traceback of embedding exception.

**Comment by ai408** [comment]
@yuzhichang Thank you for your reply. the errrors as follow:
```
Traceback (most recent call last):
  File "/mnt/f/ragflow/ragflow/rag/svr/task_executor.py", line 403, in do_handle_task
    tk_count, vector_size = embedding(cks, embd_mdl, r["parser_config"], callback)
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/f/ragflow/ragflow/rag/svr/task_executor.py", line 307, in embedding
    vts, c = mdl.encode(cnts[i: i + batch_size])
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/f/ragflow/ragflow/api/db/services/llm_service.py", line 209, in encode
    emd, used_tokens = self.mdl.encode(texts, batch_size)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/f/ragflow/ragflow/rag/llm/embedding_model.py", line 613, in encode
    np.array([d["embedding"] for d in res["data"]]),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: 'NoneType' object is not iterable
```

**Comment by yuzhichang** [comment]
@ai408 @Linnnkkk Fixed by #3792
